pkg/logs/build_logs.go

```diff
@@ -122,24 +122,20 @@ func (c *BuildLogsClient) watchReadyContainers(ctx context.Context, readyContain
 
 			switch r.Type {
 			case watch.Added, watch.Modified:
-
```

Comment:
It is a bit unclear to me what happens without these changes. 
Does it tail logs out of order? Tail only the isitio logs?

Reply:
Both. It will tail logs out of order if any other container is ready before prepare. It will also tag every container, even sidecars